examples/stocks/lib/i18n/regenerate.md

```diff
@@ -1,10 +1,19 @@
-To rebuild the i18n files:
+## Regenerating the i18n files
 
+The files in this directory are based on ../lib/stock_strings.dart which defines all of the localizable strings used by the stocks app. The stocks app uses the [Dart `intl` package](https://github.com/dart-lang/intl).
+
+Rebuilding everything requires two steps.
+
+With the `examples/stocks` as the current directory, generate `intl_messages.arb` from `lib/stock_strings.dart`:
 ```
-pub run intl_translation:generate_from_arb \
-  --output-dir=lib/i18n \
-  --generated-file-prefix=stock_ \
-  --no-use-deferred-loading \
-  lib/*.dart \
-  lib/i18n/*.arb
+flutter pub pub run intl_translation:extract_to_arb --output-dir=lib/i18n lib/stock_strings.dart
```

Comment:
just to clarify, my suggestion only changes one word in this file, i'm not suggesting changing the flutter tool. we already support "flutter packages pub", that's actually the canonical way to say it, "flutter pub pub" is just an alias.